src/Temperature.cc

```diff
@@ -160,6 +160,18 @@ Temperature Temperature::operator+(const Temperature &_temp)
   return this->dataPtr->kelvin + _temp;
 }
 
+/////////////////////////////////////////////////
+Temperature Temperature::operator+(const double _temp) const
+{
+  return this->dataPtr->kelvin + _temp;
+}
+
+/////////////////////////////////////////////////
+Temperature Temperature::operator+(const Temperature &_temp) const
+{
+  return this->dataPtr->kelvin + _temp;
```

Comment:
```suggestion
  return this->dataPtr->kelvin + _temp.Kelvin();
```

I think this may be needed